[2022-07-22 12:59:56,523] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: tutorial.t2 scheduled__2022-06-16T00:00:00+00:00 [queued]>
[2022-07-22 12:59:56,688] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: tutorial.t2 scheduled__2022-06-16T00:00:00+00:00 [queued]>
[2022-07-22 12:59:56,695] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-07-22 12:59:56,697] {taskinstance.py:1377} INFO - Starting attempt 1 of 2
[2022-07-22 12:59:56,703] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-07-22 12:59:56,893] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): t2> on 2022-06-16 00:00:00+00:00
[2022-07-22 12:59:56,912] {standard_task_runner.py:52} INFO - Started process 760 to run task
[2022-07-22 12:59:56,932] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'tutorial', 't2', 'scheduled__2022-06-16T00:00:00+00:00', '--job-id', '33', '--raw', '--subdir', 'DAGS_FOLDER/nickdag.py', '--cfg-path', '/tmp/tmpa103twtt', '--error-file', '/tmp/tmpw6gpb00t']
[2022-07-22 12:59:56,938] {standard_task_runner.py:80} INFO - Job 33: Subtask t2
[2022-07-22 12:59:57,044] {logging_mixin.py:115} WARNING - /home/***/.local/lib/python3.7/site-packages/***/configuration.py:528 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2022-07-22 12:59:57,632] {task_command.py:371} INFO - Running <TaskInstance: tutorial.t2 scheduled__2022-06-16T00:00:00+00:00 [running]> on host 2f63307804d5
[2022-07-22 12:59:58,158] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=caroline.kosalka@gmail.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=tutorial
AIRFLOW_CTX_TASK_ID=t2
AIRFLOW_CTX_EXECUTION_DATE=2022-06-16T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-06-16T00:00:00+00:00
[2022-07-22 12:59:58,163] {python.py:173} INFO - Done. Returned value was: goodbye
[2022-07-22 12:59:58,384] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=tutorial, task_id=t2, execution_date=20220616T000000, start_date=20220722T125956, end_date=20220722T125958
[2022-07-22 12:59:58,485] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-07-22 12:59:58,790] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-07-25 09:17:58,296] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: tutorial.t2 scheduled__2022-06-16T00:00:00+00:00 [queued]>
[2022-07-25 09:17:58,358] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: tutorial.t2 scheduled__2022-06-16T00:00:00+00:00 [queued]>
[2022-07-25 09:17:58,360] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-07-25 09:17:58,362] {taskinstance.py:1377} INFO - Starting attempt 1 of 2
[2022-07-25 09:17:58,363] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-07-25 09:17:58,412] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): t2> on 2022-06-16 00:00:00+00:00
[2022-07-25 09:17:58,424] {standard_task_runner.py:52} INFO - Started process 720 to run task
[2022-07-25 09:17:58,439] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'tutorial', 't2', 'scheduled__2022-06-16T00:00:00+00:00', '--job-id', '32', '--raw', '--subdir', 'DAGS_FOLDER/nickdag.py', '--cfg-path', '/tmp/tmpzj02unxm', '--error-file', '/tmp/tmppaosq50f']
[2022-07-25 09:17:58,443] {standard_task_runner.py:80} INFO - Job 32: Subtask t2
[2022-07-25 09:17:58,515] {logging_mixin.py:115} WARNING - /home/***/.local/lib/python3.7/site-packages/***/configuration.py:528 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2022-07-25 09:17:58,690] {task_command.py:371} INFO - Running <TaskInstance: tutorial.t2 scheduled__2022-06-16T00:00:00+00:00 [running]> on host 01e6b1df1873
[2022-07-25 09:17:58,802] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=caroline.kosalka@gmail.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=tutorial
AIRFLOW_CTX_TASK_ID=t2
AIRFLOW_CTX_EXECUTION_DATE=2022-06-16T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-06-16T00:00:00+00:00
[2022-07-25 09:17:58,805] {python.py:173} INFO - Done. Returned value was: goodbye
[2022-07-25 09:17:58,860] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=tutorial, task_id=t2, execution_date=20220616T000000, start_date=20220725T091758, end_date=20220725T091758
[2022-07-25 09:17:58,926] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-07-25 09:17:58,981] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
